projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/aux-key.e2e-spec.ts

```diff
@@ -0,0 +1,270 @@
+import { testProductUrl } from '../../helpers/accessibility/tabbing-order';
+
+context('Auxiliary Keys', () => {
+  describe('Category Navigation', () => {
+    before(() => {
+      cy.server();
+      cy.route(
+        `${Cypress.env('API_URL')}/rest/v2/electronics-spa/cms/components*`
+      ).as('getComponents');
+      cy.visit('/');
+      cy.wait('@getComponents');
+    });
+
+    it('should open menu with space key', () => {
+      cy.get('cx-page-slot[ng-reflect-position="NavigationBar"]').within(() => {
```

Comment:
we shouldn't rely on angular-related attributes. let's try to grab this element in a different way

Reply:
also applies to other lines of this file